polaris-core/src/main/java/org/apache/polaris/core/storage/StorageLocation.java

```diff
@@ -19,8 +19,9 @@
 package org.apache.polaris.core.storage;
 
 import jakarta.annotation.Nonnull;
-import java.net.URI;
+import org.apache.hadoop.yarn.logaggregation.LogCLIHelpers;
```

Comment:
Yarn??

Reply:
Yeah...no idea why IntelliJ did this to me 😅 thanks for catching!